sillsdev/web-languageforge
# Update field names to be consistent with FLEx

### Fixes #1730 

## Description

| LF labels | FLEx labels |
| --------- | ----------- |
| Word       | Lexeme Form |
| Meaning | Sense              |
| Part of Speech | Grammatical Category |
| Type | Sense Type |

## Checklist

- [x] I have updated tests that prove my fix is effective or that my feature works

## Testing

Testers, use the following instructions against our staging environment. Post your findings as a comment and include any meaningful screenshots, etc.

- Ensure field names referenced in table above have changed to the correct label
- Ensure there are no more old field name references in the project configuration area
